### PR TITLE
Fat 888

### PR DESCRIPTION
**Pull request by jakhongirabdukhamdiov**
Implement e-2-e automation of test case C546: Create a new request for the "Page" type.

**REF:** https://issues.folio.org/browse/FAT-888

### CODE REVIEW

File: cypress/support/fragments/inventory/createPageTypeRequest.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+const actionsButton = Button('Actions');
+const saveAndCloseButton = Button({ id: 'clickable-save-request' });
+const newRequestButton = Button('New Request');
+const selectUserModal = Modal('Select User');
+const loanAndAvailabilitySection = Section({ id: 'acc06' });
+const inventorySearch = TextField({ id: 'input-inventory-search' });
+const searchButton = Button('Search');
+
+function deleteRequestApi(requestId) {
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/createPageTypeRequest.js:26]: please use the same function from cypress/support/fragments/requests/requests.js (we already have function like this)

File: cypress/support/fragments/inventory/createPageTypeRequest.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+          permanentLocationId: instanceRecordData.permanentLocationId,
+          sourceId: instanceRecordData.sourceId,
+        }],
+        items: [[item]],
+      });
+      cy.wrap({ item, instanceRecordData, materialTypeValue });
+    });
+  },
+
+  findAndOpenInstance(instanceTitle) {
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/createPageTypeRequest.js:97]: please use already created function

File: cypress/support/fragments/inventory/createPageTypeRequest.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+    cy.do([
+      inventorySearch.fillIn(instanceTitle),
+      searchButton.click(),
+      MultiColumnListCell({ row: 0, column: instanceTitle, columnIndex: 1 }).click(),
+    ]);
+    cy.expect(Section({ id: 'pane-instancedetails' }).exists());
+  },
+
+  openHoldingsAccordion(holdingId) {
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/createPageTypeRequest.js:106]: please use already created function

File: cypress/support/fragments/inventory/createPageTypeRequest.js [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+  selectUser(username) {
+    cy.do([
+      TextField({ name: 'query' }).fillIn(username),
+      Button('Search').click(),
+      MultiColumnListCell({ row: 0, column: username }).click(),
+    ]);
+  },
+
+  selectPickupServicePoint() {
+    cy.do(cy.get('[name="pickupServicePointId"]').select('Circ Desk 1'));
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/createPageTypeRequest.js:130]: please use interactors

File: cypress/support/fragments/inventory/createPageTypeRequest.js [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+      const count = +el.parentElement.querySelector('a').textContent;
+      expect(count).to.be.greaterThan(0);
+    }));
+  },
+
+  verifyUserRecordPage(username) {
+    cy.expect(Heading({ level: 2, text: including(username) }).exists());
+  },
+
+  verifyRequestsPage() {
```
Comment (TetianaParanich) [inline cypress/support/fragments/inventory/createPageTypeRequest.js:169]: please move to related fragment

**Review by MaksimTerekhov — APPROVED** [review]

**Pull request merged**